apps/web/src/components/UserProfile/ProfileSections/AboutSection.tsx

```diff
@@ -219,28 +219,6 @@ const AboutSection: React.FC<AboutSectionProps> = ({
           ""
         )}
       </div>
-      {!firstName &&
```

Comment:
Was this intentional?  It seems out of scope.

Reply:
This section had two null states, styled totally differently from each other, that were both showing at the same time. The redundancy seemed enough for me to remove it as a part of my clean up, but if it was doing something specific that I'm missing, we can absolutely restore it.

Reply:
![image](https://user-images.githubusercontent.com/6960386/221990637-e48af5d1-0bcd-42ef-96a6-fd2394e7583f.png)